src/vivarium/framework/state_machine.py

```diff
@@ -353,11 +356,8 @@ def __iter__(self):
     def __len__(self):
         return len(self.transitions)
 
-    def __str__(self):
-        return str([str(x) for x in self.transitions])
-
     def __repr__(self):
-        return repr([repr(x) for x in self.transitions])
+        return f"TransitionSet(transitions={[x for x in self.transitions]})"
```

Comment:
I think I'm still a little confused about why sometimes we have just repr and sometimes both repr and str. I haven't been able to figure out the pattern - e.g., here we only have a repr with information about the specific transition set but machine below has both a repr with info about the specific machine and str with just Machine?

Reply:
I'll remove the str from Machine

Reply:
that was an oversight